### ISSUE TITLE
Hi, How can I apply it into maxpooling2d?

### ISSUE THREAD

**Issue by adzhua** [CyberZHG/keras-piecewise-pooling#1]

**Comment by CyberZHG** [comment]
Are you working with spatial data like images?  I think resizing the images to a fixed size would be a better choice than piecewise pooling.

**Comment by adzhua** [comment]
@CyberZHG thank you, how can I concatenate the PiecewisePooling1D outputs? when i try it like this:
pool_layer = Concatenate(axis=0)([pool_layer[0], pool_layer[1]])
I've got the following error: 'NoneType' object has no attribute '_inbound_nodes'

**Comment by CyberZHG** [comment]
```python
import keras
import keras.backend as K
import numpy as np
from keras_piecewise_pooling import PiecewisePooling1D


data = [
    [1, 3, 5, 2],
    [1, 3, 5, 2],
]
positions_1 = [
    [2, 4],
    [3, 4],
]
positions_2 = [
    [3, 4],
    [2, 4],
]

data_input = keras.layers.Input(shape=(None,))
position_1_input = keras.layers.Input(shape=(len(positions_1),), dtype='int32')
position_2_input = keras.layers.Input(shape=(len(positions_2),), dtype='int32')
pool_1 = PiecewisePooling1D()([data_input, position_1_input])
pool_2 = PiecewisePooling1D()([data_input, position_2_input])
pool_layer = keras.layers.Concatenate()([pool_1, pool_2])
model = keras.models.Model(inputs=[data_input, position_1_input, position_2_input], outputs=pool_layer)
model.compile(optimizer=keras.optimizers.Adam(), loss=keras.losses.mean_squared_error)
model.summary()
print(model.predict([np.asarray(data), np.asarray(positions_1), np.asarray(positions_2)]))
```
